### DISCUSSION TITLE
Moving game from 'low entrance fee' to 'zero entrance fee' for mass adoption

### DISCUSSION THREAD

**Discussion by drumdrumdrum3337** [sunflower-land/sunflower-land#1371] [Ideas]
This is somewhat related to this discussion - https://github.com/sunflower-land/sunflower-land/discussions/1173

I have a paranoid theory that LE won't bring in as many new users as we'd like. This is because:

a) crypto is not as hot atm, because we are in the bear market that could be here for several months to come, and 
b) I have a sense that there are not that many crypto gamers that are not yet aware of SFL, 

Hence, the game has to rely on the non-crypto gamers to grow. This could be problematic, as entering a web3 game has a steep learning curve. Main friction points are:

- understanding and setting up a crypto wallet (installing browser add-ons and phone apps, adding polygon, etc);
- buying matic somewhere and sending it to the wallet (not all the countries support direct purchase of crypto in metamask).

Regardless of how low the fee for minting a farm is, the necessary steps to do so might deter a lot of users. Having an option to sign up with an email, and then gamify the process of creating and adding a wallet to the game, is indeed a good first step, but it doesn't remove the frictions completely, e.g. the need to purchase matic for the fiat money in order to start playing.

**Proposal** 

TL;DR: Move away from the idea of low-entrance fee game to the idea of free-entrance game by removing the farm minting fee completely. After registering with their email, new users should be furnished with a new farm and a basic set of resources in order to start earning their first crypto (SFL) by farming. Once they gather enough SFL, we should prompt them to withdraw their SFL to their new wallet, do their first swap for matic, and then perform their first sync on the blockchain (exciting!). 

**Rationale**
By dropping the low entry fee, we minimize both aforementioned friction points, since game won't require any crypto to start with. Moreover, it will allow new players to earn their first crypto just by playing (and set up all the technical tools in the process). This could be marketed as the game that let's you in into the world of crypto with no tech background and 0 investments.

**Important notes:**
- Revenue from the farm minting fee would cease to exist. This could be partially mitigated by increasing syncing fee for a certain period of time for those players who mint their farm for free. Also, assumption is that the free farm mint will attract significantly larger number of users than the low-fee mint. More players would perform more syncs hence offset the revenue lost bcs of the 0 fee to mint.

- potential abuse by bots trying to exploit free farms and the initial set of resources. Could be mitigated by certain withdrawal limitations.

- it could be just too late to change anything this close to the LE.

Thanks for reading and I'd happy to just take a pill for the paranoia to go away.

**Comment by adamhannigan** [comment]
@drumdrumdrum3337 Thanks for raising this 😄 

A big purpose of the mint fee is to deter multi-accounters and botters from minting free farms and collecting SFL. 

Without a $5 fee, there is essentially nothing these accounts will lose if they are banned. The $5 is useful to deter these people, since they will lose the $5 if they are banned.

Without an upfront fee, it is a very tempting target for multi-accounters.

We are currently planning a free version of the game where non crypto users can begin playing for a couple of hours. To expand their land they must then pay the mint fee for the account.

**Reply by drumdrumdrum3337** [reply]
@adamhannigan Thanks for your response! I am curious if you have an idea how effective this particular step is in deterring multi-accounters? 

Just thinking out loud: what if the new farms can withdraw any resources, apart from the sfl needed for syncing, only after playing for some period of time, like 3 months or so? The time before the withdrawal would be the time for other multi-account prevention systems to flag any cheaters. 

Assumption is that this might have the same effect on cheaters as the minting fee right now: now they need to gather around 80sfl to be net negative, if the price of sfl goes up, then even less than that. Everything on top of that is their profit, unless we catch them earlier. 
If they are forced to play fairly for some months, the result might be roughly the same for the economy, but we’d gain the opportunity to onboard non-crypto folks so much easier. Technically, they could mint an unlimited number of farms if only email is required, but then they would have to play manually for months, hopefully this would be deterring enough.

I see that cheaters are likely the reason why most of the crypto games sell starter packs in one form of the other. 

I also understand that you have thought about this a lot, and might not want to experiment with your game in this regard anymore. Apologies for dragging the discussion! It just hurts to see that cheaters are such a significant factor impeding the accessibility of the whole industry to the non-crypto players.

**Reply by adamhannigan** [reply]
@drumdrumdrum3337 That is an interesting idea around the 3 months delay for withdrawals. This definitely would deter multi-accounters. I can also see it being perceived a bit as "freezing" assets as well.

No one in Crypto has perfected the model yet, so we are always open to experimenting with these concepts.

The 3 month delay would need to apply for new farms or farms that are transferred (As bots often buy/flip farms from Open Sea).

I wonder if a time limit is useful in this case or if it would be better to use an in-game milestone. For instance, once they harvest X crops they unlock the ability to withdraw from the bank.

It could be tricky as well if a player deposits SFL or items onto their farm. Ideally we don't want to lock these people from withdrawing those same assets out 🤔 

What are your thoughts on these issues?

**Reply by sir-canada** [reply]
this is an interesting idea! just saw this and wanted to add my two cents :grin: 

could easily have a counter that any value added to a farm within the 3 month period, sets a 'deposited/withdraw' value.

deposit 50 sfl into farm:
value is now 50
withdraw 25 sfl from farm:
value is now 25
deposit another 50 sfl
value is now 75
max withdraw amount is now 75

regarding NFTs, could easily add a 'deposited' list. 
deposit a 422 422 426
deposited nft list: 422, 422, 426.
withdraw nft 422
deposited nft list: 422, 426
etc.

**Reply by drumdrumdrum3337** [reply]
@adamhannigan perception of 'freezing' assets is indeed better to be avoided since that is not an ultimate goal. There should be a way to make this harvest withdrawal unlock a part of the gameplay.

**Example:** 

- there's a Farmers' Trading Union - its members can withdraw and trade. 
- However, in order to become a member of the Union a new farmer has to prove that he/she can produce products of the expected quality. 
- So here comes the quest: a new farmer has to collect a certain amount of each resource in order to obtain that resource's badge (e.g. Potato badge, Fishing badge, etc - in-game milestones). 
- Once all the required resource badges are collected, they could be exchanged for the Trading Union badge that allows trading and withdrawing. 
- New farmers are welcome to add boosting nfts purchased on OS to their farms in order to complete the quest quicker. So only nfts are moveable, not the resources. Not sure they would need to deposit any SFL at this point.
- Once Trading Union badge is obtained, player has just enough seeds left to earn their first SFL that is to be swapped for matic in order to perform player's first sync on chain.
- If the accomplishment is accompanied by some minor experience boosting nft, then the old farmers, who are in the Union by default, can also participate in the quest to earn that NFT.

**Important notes:** 
- there should be enough seeds available on the new farm to pass the quest without a need for a sync. Otherwise, it goes against the idea of 'earning first crypto without the fiat investments'. 
- Such quest would probably mean that the collection of all the resources happens in the game session and is not synced on the blockchain. Only the Trading Union badge makes it to the blockchain once exchanged for all the required resource badges. 

Concern:
- Would this make the onboarding to the game too long and/or complicated?

**Reply by alphacruze** [reply]
How would user go about swapping to matic? After all Polygon is Decentralized, so swapping has the requirement of needing matic

Unless you go through the "zero-gas fee" method, however that isn't possible with every token

**Reply by drumdrumdrum3337** [reply]
that's a good point, I don't know. This is pretty much where my technical knowledge ends.

**Reply by alphacruze** [reply]
Hmm, sorry I only saw your reply now

I think the best way would be, take the users to a Faucet, perhaps one that is funded by the sunflower-land community for the sunflower-land community, like only being accessible in-game (The garden) or something

The faucet would release enough for 3 transactions or so, this would give the person enough gas to swap from SFL to matic and have some left over so they can cover the gas along with synching fee

Perhaps a small portion of synching fee, like 1% of it could go into the faucet even as an option

**Reply by adamhannigan** [reply]
Some great points. The team has been discussing this idea over the past week.

I view free-to-play as the direction the game needs to go, but it will require some significant work. 

A huge part of making it free to play means onboarding non Web3 natives. This means that we need to simplify the withdrawing/depositing flow to something that is extremely basic. We will want to encourage people to deposit SFL into their farms (to get a head start) but the withdraw milestones/locking is a user experience challenge that needs to be solved.

We won't take decisive action on this idea right now but it is something we want to pursue. Over the coming months we need to formulate solutions for the UX, technical, antibotting & onboarding challenges that free to play will bring on

**Reply by drumdrumdrum3337** [reply]
makes total sense, Adam, and I completely realize this is far from a trivial fix. Thanks for sharing your thoughts on free-to-play concept, and since the most immediate goal of this discussion was, well, to initiate a discussion, we can consider it closed (there doesn't seem to be an option to mark discussion as fix/closed/resolved so just leaving it as is).

**Comment by Formalinfarmer** [comment]
I would like to express some concern over the "Energy" or "stamina" idea for LE.  This is a ploy used in mobile games to generate money from players that do not want to wait to play.  My major concern is that this game currently already makes you wait to play due to the time to grow crops or re-grow a tree or mining veins.  If the new game makes me spend $/resources to recharge my bumpkin and also makes me wait for crops to grow etc. I will very quickly lose interest as I do with any mobile game.  I really hope that it will be very easy to significantly increase the bumpkin energy meter early in the game or I see disaster as those that are used to only waiting for crop grow times or tree/stone/iron/gold to refresh for additional mining now also having to feed a bumpkin to even have the chance to wait for crops and resources to be available again.  I am very worried about this.  I don't want this game to become like another useless mobile game where you only get to play more than 30 min a day if you are willing to pay.  PLEASE do not go in this direction.

**Reply by alphacruze** [reply]
I'm not sure if you've watched the streams about the energy system or perhaps things have changed since the last time I have had time to watch a stream.

However, based on what was discussed and coded 2 or 3 weeks ago, energy requires extremely quickly for low level players, I think it was backup to 100% within an hour and only gradually takes longer as you rise in the levels, however the maximum amount of time it takes to recover 100% stamina is 24 Hours and that's at extremely high level where you have an insane amount of stamina... well relatively speaking.

As an example(I can't remember the figures anymore...) if a level 1 Bumpkin has 10 stamina, then within 1 hour that 10 stamina will be recovered, now lets say each level up increases stamina by 10 then level 10 bumpkin will have 100. This bumpkin will recover stamina faster than the level 1,but maybe only 25 per hour, so in 4 hours = 100% stamina.

So now we look at a theoretical level 100 bumpkin, has 1000 Stamina, if emptied and left alone for 24 hours, all 1000 will be recovered so that's recovering 41.6/ 42 stamina per hour.
These are just examples not actual figures. But I'm sure you get the idea, Stamina recovers very quickly
